packages/components/src/filters/advanced/number-filter.js

```diff
@@ -68,13 +81,15 @@ class NumberFilter extends Component {
 					className="woocommerce-filters-advanced__input"
 					type="number"
 					value={ value }
+					aria-label={ label }
 					onChange={ onChange }
 				/>
 				: <TextControlWithAffixes
 					prefix={ <span dangerouslySetInnerHTML={ { __html: currencySymbol } } /> }
 					className="woocommerce-filters-advanced__input"
 					type="number"
 					value={ value }
+					aria-label={ label }
 					onChange={ onChange }
```

Comment:
Looks like you can add a prop `min="0"` to prevent negative numbers.

Reply:
Let's handle this in a separate PR - the `NumberFilter` component doesn't have any sort of input validation yet.